### DISCUSSION TITLE
Rotas para acesso aos índices do IBOVESPA

### DISCUSSION THREAD

**Discussion by msalvexx** [BrasilAPI/BrasilAPI#239] [Ideas]
Existem alguns serviços que fornecem os índices dos papéis IBOVESPA, mas a maioria, se não todos são pagos. No entanto creio que essa é uma informação que deveria ser acessível a todos os tipos de investidores, desde investidores iniciantes até a grandes empresas.

Acho importante também levantar certos questionamentos:
- A API pode ser disponibilizada em tempo real? 
- Limitação de requisições por período?
- Disponibilização de webhooks para cadastro de eventos de preço?

**Comment by HelderSi** [comment]
Essa API seria muito bem vinda. Tenho interesse em contribuir nessa parte.

**Comment by CWojahn** [comment]
Ótima ideia, gostaria de contribuir nessa parte.

**Comment by marcelotq** [comment]
Também gostaria de contribuir, contem com meu apoio.

**Comment by HelderSi** [comment]
Vcs têm alguma API para usar como referência? todas são pagas mesmo?

**Comment by CWojahn** [comment]
Fiz algumas buscas porem API's gratuitas são todas com delay de 15min.
Até onde tenho de informação a B3 cobra para fornecer os dados em real time.
Abaixo tem a API da Alpha Vantage no site não fala nada sobre o delay e não consegui verificar ainda se é real time ou não mas tem limite de request por dia no plano gratuito.
https://www.alphavantage.co/documentation/

a B3 tem o site abaixo mas com delay de 15min e ainda teria que fazer scrap dos dados e rodar um robo fulltime pra rodar em cima de todos os ativos, o que poderia acabar gerando bloqueio de IP em função do excesso de requisições.
http://www.b3.com.br/pt_br/market-data-e-indices/servicos-de-dados/market-data/cotacoes/
